samples/FunctionApp/FunctionApp.csproj

```diff
@@ -14,12 +14,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.ApplicationInsights" Version="1.0.0-preview3" />
```

Comment:
Once this has flowed to this extension, I'll update the tests to ensure it all keeps working e2e.